samples/streams/build.sbt

```diff
@@ -0,0 +1,7 @@
+name := "iteratee-sample"
```

Comment:
```suggestion
name := "streams-sample"
```

Reply:
Good catch. 

This was a cherry-pick from master, I will change there as well.